job-search-web-scraping.py

```diff
@@ -1,39 +1,88 @@
+import smtplib
+import sys
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from os.path import basename
+from pathlib import Path
+
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
-from pathlib import Path
 
-import sys
+from config import cfg
+
+
+def send_email(sender_email_address, email_password,
+               receiver_email_address, email_subject,
+               email_body, file_path):
+    """
+    sender_email_address: sender email address
+    email_password: sender email password
+    receiver_email_address: receiver email password
+    email_subject: email subject
+    email_body: email body
+    file_path: file to attach in the email
+    """
+    email_smtp = "smtp.gmail.com"
```

Comment:
The email_smtp can be made configurable in the config file. This will allow other mail servers to be used.